flash/data/callback.py

```diff
@@ -0,0 +1,67 @@
+from typing import Any, List, Sequence
+
+from pytorch_lightning.callbacks import Callback
+from pytorch_lightning.trainer.states import RunningStage
+from torch import Tensor
+
+
+class FlashCallback(Callback):
+
+    def on_load_sample(self, sample: Any, running_stage: RunningStage) -> None:
+        """Called once a sample has been loaded."""
+
+    def on_pre_tensor_transform(self, sample: Any, running_stage: RunningStage) -> None:
+        """Called once an object has been transformed."""
+
+    def on_to_tensor_transform(self, sample: Any, running_stage: RunningStage) -> None:
+        """Called once an object has been transformed to a tensor."""
```

Comment:
minor documentation detail, but might be confusing to call in the same way cast an object to a torch.tensor as a sequence of data transforms.

Reply:
Changed to """Called once ``to_tensor_transform`` have been applied to a sample."""